torch/_windows.py

```diff
@@ -0,0 +1,30 @@
+from typing import Optional
+
+import torch.signal.windows
+
+__all__ = ["kaiser_window"]
+
+
+def kaiser_window(
```

Comment:
My suggestion is not to worry about `torch.kaiser_window` in this PR.

A follow-up PR will let us focus on the `torch.signal.windows.kaiser` logic, and `torch.kaiser_window` isn't really an alias for `torch.signal.windows.kaiser` because of the `window_length` and `periodic` arguments having different names

Reply:
Ok, I'll undo this change. Just for the record, the alias we'd like to have the same signature as the old one, correct?

Reply:
Yes. More precisely it would be a "near alias" that forwards its arguments to this implementation